packages/stateless/src/lib/provider/provider.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum CheckIntegrationResponseEnum {
+  INVALID_EMAIL = 'From Email id is invalid/incorrect',
```

Comment:
When doing the error messages enum, I would stick just to an enum value. The display text should be handled in the UI or the server should return the message directly.

```suggestion
  INVALID_EMAIL = 'invalid_email',
```

My suggestion is just to have the enum value here and separate the responsibilities over this one. 

Let's discuss it together later :)